src/components/formFields/select/select.tsx

```diff
@@ -72,7 +73,25 @@ export const SelectMultiple = ({
   hideErrorMessage,
   menuPlacement,
   placeholder,
+  ariaLabel,
 }: ISelectProps & IReactHookFormProps): JSX.Element => {
+  React.useEffect(() => {
+    document.querySelectorAll('[id*="live-region"]').forEach((element: any) => {
+      if (element?.role !== "presentation") {
+        element.setAttribute("role", "presentation");
+      }
+    });
+    document.querySelectorAll('[class*="indicatorSeparator"]').forEach((element: any) => {
+      if (element.role !== "presentation") {
+        element.setAttribute("role", "presentation");
+      }
+    });
+    document.querySelectorAll('[class*="a11yText"]').forEach((element: any) => {
+      if (element.role !== "presentation") {
+        element.setAttribute("role", "presentation");
+      }
+    });
+  }, []);

```

Comment:
The `role="none"` does not exist (we discussed it in our call as a potential change), so we shouldn't do anything with that. The `role="presentation"` is the correct one for `a11Text` and `live-region` I think. However, we could use `role="separator"` for the `indicatorSeparator`. 

Right now, we're duplicating this block of code in each component, even though the code is identical. I've refactored it into a generic function that we can use in each of these components:

```
const setAttributes = (): void => {
  const setRoleToPresentation = (selector: string, role: string) => { // we could type this role prop
    document.querySelectorAll(selector).forEach((element) => {
      if (element.getAttribute("role") !== "presentation") element.setAttribute("role", role);
    });
  };

  setRoleToPresentation('[id*="live-region"]', "presentation");
  setRoleToPresentation('[class*="indicatorSeparator"]', "separator");
  setRoleToPresentation('[class*="a11yText"]', "presentation");
};
```

And then, instead of having this logic in each component, we can refactor the `useEffect` to this:

```
React.useEffect(() => {
  setAttributes();
}, []);
```